packages/apps/docs/src/components/Markdown/Code/Code.tsx

```diff
@@ -8,8 +8,21 @@ interface IProp {
 
 export const Code: FC<IProp> = ({ children, ...props }) => {
   if (typeof children === 'string') {
-    return <StyledInlineCode>{children}</StyledInlineCode>;
+    return <code className={inlineCode}>{children}</code>;
   }
 
-  return <StyledCode {...props}>{children}</StyledCode>;
+  return (
+    <code className={code} {...props}>
+      {React.Children.map(children, (child) => {
+        if (!React.isValidElement(child) || !child) {
+          return null;
+        }
+
+        return React.cloneElement(child, {
+          ...child.props,
+          className: codeLine,
+        });
+      })}
+    </code>
```

Comment:
Is mapping through the children still necessary?

Reply:
yes! they need to get assigned the `codeLine` class:
```typescript
export const codeLine = style([
  sprinkles({
    width: '100%',
    display: 'inline-block',
    fontFamily: '$mono',
  }),
  {
    selectors: {
      '&::before': {
        counterIncrement: 'line',
        content: 'counter(line)',
        overflowWrap: 'normal',
        display: 'inline-block',
        width: '1rem',
        marginRight: `${vars.sizes.$4}`,
        marginLeft: `${vars.sizes.$4}`,
        textAlign: 'right',
        fontSize: `${vars.sizes.$sm}`,
        color: `${vars.colors.$neutral3}`,
      },
    },
  },
]);

Reply:
Do you think this is an instance in which using globalStyle would be a better option?

Reply:
Even if the style applied with the parent class as a selector, we cannot use selectors inside `globalStyle`

Reply:
oh i see that is not possible either. Nevermind 👍